.travis.yml

```diff
@@ -1,19 +1,20 @@
 language: go
+os:
+  - linux
+  - osx
 go:
 - '1.11'
 env:
 - GO111MODULE=on
-jobs:
-  include:
-  - stage: Test and build
-    script: make
-    before_deploy:
-    - sudo apt-get install -y upx
-    - make dist-release
+script: make
+before_deploy:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get update; fi
```

Comment:
Wonder if we are actually using upx?  I see it's used in make shrink but can't see an example of us using it?  Is this why the macos build works without issue even though we don't install upx?

Reply:
Oh, yeah, I don't think we are any longer. I'll see if it actually makes any difference in the artefact size and if not remove it entirely.

Reply:
Before:

```                     
[jd@genesis-dev] ~/src/lyra % ls build
total 38M
drwxrwxr-x  2 jd jd 4.0K Dec 13 11:06 .
drwxrwxr-x 11 jd jd 4.0K Dec 13 11:05 ..
-rwxrwxr-x  1 jd jd  38M Dec 13 11:06 lyra
```

After:

```
[jd@genesis-dev] ~/src/lyra % upx build/lyra
                       Ultimate Packer for eXecutables
                          Copyright (C) 1996 - 2017
UPX 3.94        Markus Oberhumer, Laszlo Molnar & John Reiser   May 12th 2017

        File size         Ratio      Format      Name
   --------------------   ------   -----------   -----------
  39242244 ->  15378760   39.19%   linux/amd64   lyra

Packed 1 file.
[jd@genesis-dev] ~/src/lyra % make dist-release
[jd@genesis-dev] ~/src/lyra % ls build
total 29M
drwxrwxr-x  2 jd jd 4.0K Dec 13 11:07 .
drwxrwxr-x 11 jd jd 4.0K Dec 13 11:05 ..
-rwxrwxr-x  1 jd jd  15M Dec 13 11:06 lyra
-rw-rw-r--  1 jd jd  15M Dec 13 11:07 lyra.tar.gz
-rw-rw-r--  1 jd jd   84 Dec 13 11:07 lyra.tar.gz.sha256
```

Yeah, worth it. I'll add that step back in and see about making it work with macOS as well.